test/acceptance/tests/fixtures/bases/static-metrics-app/service.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: static-metrics-app
+spec:
+  selector:
+    app: static-metrics-app
+  ports:
+    - port: 80
```

Comment:
should this have targetPort: 9090 to route to the metrics app?

Reply:
it doesn't need it for the test to pass (test checks that the merged metrics port has merged metrics by directly talking to the pod) but should probably have it for completeness!